packages/panzer/disc-fe/src/evaluators/Panzer_GatherSolution_BlockedEpetra_Hessian.hpp

```diff
@@ -90,13 +90,13 @@ namespace panzer
        *  fields, sets up dependent tangent fields (if requested), and
        *  determines the first active name.
        *
-       *  \param[in] indexers The `vector` of `UniqueGlobalIndexer`s that
+       *  \param[in] indexers The `vector` of `GlobalIndexer`s that
        *                      handle the global unknown numbering.
        *  \param[in] p        A `ParameterList` used as input for
        *                      `GatherSolution_Input`.
        */
       GatherSolution_BlockedEpetra(
-        const std::vector<Teuchos::RCP<const UniqueGlobalIndexer<LO, int>>>&
+        const std::vector<Teuchos::RCP<const GlobalIndexer<LO, int>>>&
```

Comment:
Are you compiling with Hessian support? Do we have a templated global indexer as well as a non-templated form?

Reply:
No. Hessian support has been broken for a long time. It is experimental and currently no one is using it.  I may clean up and restore Hessian support in the fall if a certain project asks for it.